### PR TITLE
New param docker all

### PR DESCRIPTION
**Pull request by bthaile**
add new param to run docker all and deploy contracts with time control on. 

removed creating orders from create canned market call. 
changed docker all to use flash convenience methods, normal-all and fake-all.

### CODE REVIEW

File: scripts/one-docker-to-rule-them-all.sh [hunk 1]
```diff
@@ -6,22 +6,37 @@ cleanup() {
   echo "stopping geth docker image"
   docker kill geth
   yarn workspace @augurproject/gnosis-relay-api kill-relay
+  docker volume prune --force
+  docker system prune --force
```
Comment (robert-e-davidson3) [inline scripts/one-docker-to-rule-them-all.sh]: I worry about taking such destructive action inside a script that isn't obviously this destructive.

Comment (bthaile) [reply scripts/one-docker-to-rule-them-all.sh]: ok, I'll remove

**Comment by bthaile** [comment]
removed the prunes

**Review by robert-e-davidson3 — APPROVED** [review]
lgtm!

**Pull request merged**